plugins/module_utils/utils.py

```diff
@@ -69,7 +69,7 @@ def delete(client, path):
 
 def put(client, path, payload):
     resp = client.put(path, payload)
-    if resp.status != 201:
+    if resp.status not in (200, 201):
```

Comment:
If tessen is the only API endpoint that returns this status code, I would suggest that we leave the helper method alone and use the Sensu API client directly in the module. Weakening the method for the sake of single user feels a bit wrong.

Reply:
Good point, I was thinking more in the direction that both 200 and 201 are valid codes for `HTTP PUT` (according to the RFC specification :)), that is why I put it there. However, as you pointed out, in this API that is mostly not the case.